README.md

```diff
@@ -1,88 +1,108 @@
+# Mawaqit API
 
-# Mawaqit Api
+Mawaqit API is a REST API developed using the FastAPI framework, designed to interface with the [mawaqit.net](https://mawaqit.net) website. Mawaqit.net provides prayer times for over 8,000 mosques worldwide. This API aims to serve as a web application that fetches prayer times from mawaqit.net, delivering them in a streamlined JSON format. Unlike the current PHP-based website, which returns complete HTML pages for each request, this API focuses on delivering only the essential information.
 
-Mawaqi Api is a Rest Api for [mawaqit.net](https://mawaqit.net) using FastApi framework,
-the mawaqit.net website gives you the prayer times for more than 8000 mosques around the world,
-the idea behind this api is to create an api web app that uses the mawaqit website as data source
-to fetch prayer times and return them in json with the minimum information needed,
-the current website is using php so it's returning the whole html every get request.
+## Local Setup
 
+### Clone the Project
 
-
-## Run Locally
-
-Clone the project
+To get started, clone the repository to your local machine:
 
 ```bash
 git clone https://github.com/mrsofiane/mawaqit-api.git
 ```
 
-Go to the project directory
+Navigate to the project directory:
 
 ```bash
 cd mawaqit-api
 ```
 
-### With Python
+### Setup with Python
+
+Ensure the following libraries are installed before setting up a virtual environment:
+
+- uvicorn
+- requests
+- fastapi
+- httpx
+- pytest
+- bs4
+- redis
 
```

Comment:
I guess we dont need to specify these libraries because they are all in the requirements.txt